src/System.Collections/tests/Generic/LinkedList/LinkedList.Generic.Tests.Constructor.cs

```diff
@@ -98,7 +98,6 @@ internal LinkedList_T_Tests(LinkedList<T> collection, T[] expectedItems)
         /// <summary>
         /// Tests the initial items in the list.
         /// </summary>
-        [Fact]
```

Comment:
😕 Why was this removed?

Reply:
Thsi is a helper method inside an internal class which is invoked from test methods in another class.